vault/blog.2022.2022-05-02-self-contained-vaults-release.md.md

```diff
@@ -0,0 +1,72 @@
+---
+id: g98s3m6jft4gnb546y7i4xg
+title: 2022 05 02 Self Contained Vaults Release
+desc: ''
+updated: 1651527766161
+created: 1651526168935
+excerpt: "Vaults and workspaces stood out as a point of confusion for both new and existing users, but we are solving this with the upcoming release of Self Contained Vaults."
```

Comment:
- add yourself as an author here: `author: user.kaan`
- it currently doesn't show up in the blog but in the future, i want to do a link/embed of author profiles in the blog